### PR TITLE
Enable "find on shelf" button if no material specimens available

### PR DESCRIPTION
**Pull request by Adamik10**
#### Link to issue
https://reload.atlassian.net/browse/DDFSOEG-359

#### Description
The find on shelf button should always be enabled for physical materials such as books, CDs, etc., even though they are unavailable. 

#### Screenshot of the result
image
image

#### Checklist

- [x] My complies with [our coding guidelines](../docs/code_guidelines.md).
- [x] My code passes our static analysis suite. If not then I have added a comment explaining why this change should be exempt from the code standards and process.
- [x] My code passes our continuous integration process. If not then I have added a comment explaining why this change should be exempt from the code standards and process.

#### Additional comments or questions
Please disregard the failing "loan-list" test. Sine is working on fixing it.

### CODE REVIEW

File: src/components/availability-label/availability-labels.tsx [hunk 1]
```diff
@@ -20,7 +20,7 @@ export interface AvailabilityLabelsProps {
   cursorPointer?: boolean;
 }
 
-export const AvailabiltityLabels: React.FC<AvailabilityLabelsProps> = ({
+export const AvailabilityLabels: React.FC<AvailabilityLabelsProps> = ({
```
Comment (kasperg) [inline src/components/availability-label/availability-labels.tsx:23]: :+1: Nice to get this corrected even though it is not part of task at hand.

File: src/components/material/material-buttons/physical/MaterialButtonsFindOnShelf.tsx [hunk 2]
```diff
@@ -42,19 +42,6 @@ const MaterialButtonsFindOnShelf: FC<MaterialButtonsFindOnShelfProps> = ({
   }
 
   if (size !== "small") {
-    if (!data[0].available) {
-      return (
-        <Button
-          label={t("materialIsLoanedOutText")}
-          buttonType="none"
```
Comment (kasperg) [inline src/components/material/material-buttons/physical/MaterialButtonsFindOnShelf.tsx:49]: Please consider if fetching availability data with `useGetAvailabilityV3()` on L24 in this file can be removed now.

As far as I can tell with this change the only place the data was actually used is gone. If that is the case then the request is obsolete.

**Review by kasperg — CHANGES_REQUESTED** [review]
:+1: but I have one important consideration for you before this can be moved along.

Also please check(s) for the Find On Shelf button in our tests. The presence (or lack thereof) of buttons is something we have been going back and forth on frequently.

File: src/components/material/material-buttons/physical/MaterialButtonsFindOnShelf.tsx (hunk 2 above)
Comment (Adamik10) [reply src/components/material/material-buttons/physical/MaterialButtonsFindOnShelf.tsx:49]: That is absolutely correct. It's removed now.

**Review by kasperg — APPROVED** [review]
:+1: LGTM

**Review by jeppekroghitk — APPROVED** [review]

**Pull request merged**